learning/tour-of-beam/backend/README.md

```diff
@@ -39,6 +39,7 @@ To re-generate:
 $ go generate -x ./...
 ```
 
+> Note: this requires [`protoc`](https://grpc.io/docs/protoc-installation/) version `>= 3.15` and [`moq`](https://github.com/matryer/moq) tools to be installed
```

Comment:
Does this generate the proto stubs?  I think we should either use buf which is already used for the Beam Playground here.  Or convert the Beam Playground to this.  Either way, both should be the same.

Reply:
Created task #25775 to resolve this in a separate PR